src/frontend/src/common/components/library/Dialog/components/DialogContent/index.tsx

```diff
@@ -1,8 +1,10 @@
 import { DialogContentProps } from "@material-ui/core";
 import React from "react";
-import { StyledDialogContent } from "./style";
+import { ExtraProps, StyledDialogContent } from "./style";
 
-const DialogContent = (props: DialogContentProps): JSX.Element => {
+type Props = ExtraProps & DialogContentProps;
```

Comment:
DialogContent component now accepts an extra prop `narrow` to set width to `400px`